snapcraft/parts/lifecycle.py

```diff
@@ -170,11 +170,13 @@ def _run_in_provider(project: Project, command_name: str, parsed_args: "argparse
 
     output_dir = utils.get_managed_environment_project_path()
 
-    emit.progress("Launching build provider")
+    # FIXME: pause emitter when executing instance (needs craft-cli support)
+    emit.progress("Launching instance...")
     with provider.launched_environment(
         project_name=project.name, project_path=Path().absolute(), base=cast(str, project.base)
     ) as instance:
         try:
+            emit.message("Launched instance", intermediate=True)
```

Comment:
maybe "Instance launched"?

Reply:
It was done this way because all other messages are in the format "Installing something..." -> "Installed something", "Executing something..." -> "Executed something", but we can adjust all messages after spinner pause is implemented in craft-cli (so we can see the entire output and evaluate consistency and correctness).